rust/kcl-lib/src/engine/conn_wasm.rs

```diff
@@ -17,7 +17,7 @@ use crate::{
     execution::{DefaultPlanes, IdGenerator},
 };
 
-#[wasm_bindgen(module = "/../../src/lang/std/engineConnection.ts")]
+#[wasm_bindgen(module = "/../../src/network/connectionManager.ts")]
```

Comment:
I have made a new engine connection file. The interface is the same for rust. I am pointing it to my new class.